core/src/main/java/com/scalar/db/api/DistributedStorageAdmin.java

```diff
@@ -41,7 +41,6 @@
  * }</pre>
  */
 public interface DistributedStorageAdmin extends Admin {
-
```

Comment:
It looks like this change is unnecessary.

Reply:
Thanks, I reverted it. 
https://github.com/scalar-labs/scalardb/pull/2002/commits/ae5989603ac48d8c9f33ea1cd09b60c2db2a0f15